chrome/content/zotero/xpcom/data/tags.js

```diff
@@ -945,7 +945,14 @@ Zotero.Tags = new function() {
 			ctx.fill();
 		}
 	}
-	
+
+	// Return the first sequence of emojis from a string
+	this.extractEmojiForItemsList = function (str) {
+		// Split by anything that is not an emoji, Zero Width Joiner, or Variation Selector-16
+		// And return first continuous span of emojis
+		const re = /[^\p{Extended_Pictographic}\u200D\uFE0F]+/gu;
+		return str.split(re).filter(Boolean)[0];
```

Comment:
Maybe a little cleaner to add ` || null` here to return `null` instead of `undefined` if there's no emoji.